protocols/IPM_Medical_Group/mass_spec_sample_prep.ot2.py

```diff
@@ -0,0 +1,81 @@
+from opentrons import labware, instruments
+
+# customization
+sample_number = 48   # must be between 10 and 48
+incubation_time = 1  # in minutes
+centrifuge_time = 1  # in minutes
+
+labware.create('custom-tuberack-7ml',
```

Comment:
Here they will get an error if they run the protocol multiple times. To prevent this from happening do the following:

if container_name not in labware.list():
      ```labware.create()```